packages/patternfly-4/react-core/src/components/Popover/PopoverDialog.tsx

```diff
@@ -21,20 +26,11 @@ const PopoverDialog = ({ position, children, className, ...props }) => (
   </div>
 );
 
-PopoverDialog.propTypes = {
+export interface PopoverDialogProps extends React.HTMLProps<HTMLDivElement> {
   /** PopoverDialog position */
-  position: PropTypes.oneOf(Object.values(PopoverPosition)),
+  position?: OneOf<typeof PopoverPosition, keyof typeof PopoverPosition>;
```

Comment:
We don't need OneOf anymore. Just do `'top' | 'right'` for nice docs.

Reply:
fixed